build.go

```diff
@@ -0,0 +1,179 @@
+//+build ignore
+
+package main
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"github.com/urfave/cli"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+)
+
+var packages = []string{"cli", "altsrc"}
+
+func main() {
+	app := cli.NewApp()
+
+	app.Name = "builder"
+	app.Usage = "Generates a new urfave/cli build!"
+
+	app.Commands = cli.Commands{
+		cli.Command{
+			Name:   "vet",
+			Action: VetActionFunc,
+		},
+		cli.Command{
+			Name:   "test",
+			Action: TestActionFunc,
+		},
+		cli.Command{
+			Name:   "gfmrun",
+			Action: GfmrunActionFunc,
+		},
+		cli.Command{
+			Name:   "toc",
+			Action: TocActionFunc,
+		},
+		cli.Command{
+			Name:   "generate",
+			Action: GenActionFunc,
+		},
+	}
+
+	err := app.Run(os.Args)
```

Comment:
Perhaps the default behaviour for this (no arguments), would be to run all of these steps, or just run a build?

Reply:
that is up for discussion. this is just a port of the old python script. we can change it to what we feel would be the best

Reply:
Agree, feel free to ignore me.